packages/offix-server/package.json

```diff
@@ -31,5 +31,8 @@
   "publishConfig": {
     "access": "public"
   },
-  "gitHead": "c52429fc496cfdbaaf06fd55a509f14b12c7ce1c"
+  "gitHead": "c52429fc496cfdbaaf06fd55a509f14b12c7ce1c",
+  "dependencies": {
+    "offix-cache": "^0.1.0"
```

Comment:
Do we need that here?

Reply:
nope, i was messing with lerna it must have added it. will remove